client/public/clientJS/passwordReset.js

```diff
@@ -49,7 +49,13 @@ function getHostName() {
             return "http://localhost:8081";
```

Comment:
I think we can just use process.env.BASE_URL here which will always point to the right domain

Reply:
I can not use that in frontend though.